### DISCUSSION TITLE
Thank you...

### DISCUSSION THREAD

**Discussion by thomwinans** [community/community#126089] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Thank you very much for your careful elaboration of what you've done to use SBCL as a custom runtime for AWS Lambda.

I am refreshing with Lisp and thought it would be good to use CL as a Lambda in addition to however else I would, so I took your good work and careful explanation and modified it slightly to update for AmazonLinux 2023. A few library versions needed to be updated, and a few gaps between AWS's packaging in AL2 vs AL2023 (ssl things) had to be addressed. I attach the Dockerfile here...

Again... many thanks... I saw one solution that depended on ASDF more to gen the bootstrap core, and I found I could track and develop my understanding better with your solution.
[Dockerfile.txt](https://github.com/community/community/files/15462132/Dockerfile.txt)

All best!

**Comment by ebndev** [comment]
Hi @thomwinans,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from a specific project. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

**Comment by ebndev** [comment]
Hi again @thomwinans!

If you don't mind, would you mind letting us know how you initially ended up in the [Profile category](https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+) when creating your discussion and providing the link to the project's repository? We are trying to investigate if there is a bug or link that may be re-directing users from the project's Discussions board, if that’s where you intended to post.

Any insight would be helpful, thanks!

**Reply by ebndev** [reply]
Never mind, found the issue! Thanks again for your time and being a part of the GitHub Community 😄

**Comment by thomwinans** [comment]
Good deal – sorry to be late getting back.


Tom Winans | Managing Director
(m) +1 303 596 0601<tel:+13035960601> | (e) ***@***.***



From: ebndev ***@***.***>
Date: Tuesday, June 18, 2024 at 13:29
To: community/community ***@***.***>
Cc: Tom Winans ***@***.***>, Mention ***@***.***>
Subject: Re: [community/community] Thank you... (Discussion #126089)
You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

Never mind, found the issue! Thanks again for your time and being a part of the GitHub Community 😄
